packages/uploadthing/src/internal/handler.ts

```diff
@@ -67,13 +67,17 @@ const withExponentialBackoff = async <T>(
     backoffMs = Math.min(MAXIMUM_BACKOFF_MS, backoffMs * 2);
     backoffFuzzMs = Math.floor(Math.random() * 500);
 
-    console.error(`[UT] Call unsuccessful after ${tries} tries. Retrying in ${Math.floor(backoffMs / 1000)} seconds...`)
+    console.error(
```

Comment:
We should probably not log this til 3 or 4 tries in, not necessary for this pass